net/openvpn/patches/101-Fix-EVP_PKEY_CTX_-compilation-with-wolfSSL.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/openvpn/crypto_openssl.c
++++ b/src/openvpn/crypto_openssl.c
+@@ -51,7 +51,7 @@
+ #include <openssl/rand.h>
+ #include <openssl/ssl.h>
+ 
+-#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && !defined(LIBRESSL_VERSION_NUMBER)
++#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && !defined(ENABLE_CRYPTO_WOLFSSL) && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
I changed it like this because it was done the same way here for LibreSSL:
https://github.com/OpenVPN/openvpn/commit/4cf01c8e4381403998341aa32f79f4bf24c7ccb1

Reply:
But it's no problem to drop this hunk. I'll update the patch tomorrow.

Reply:
OK.

Reply:
Hmm... shouldn't we upstream this patch? Why do we need to carry this here in our repository and refresh it each time once there is a new version?

Reply:
Since I don't use WolfSSL myself and had only compiled it so far, I didn't want to send the patches upstream right away. But if I get positive feedback here, I can try to send these patches upstream.